src/game.cpp

```diff
@@ -1873,6 +1933,18 @@ bool Game::startup(bool *kill,
 
 	smgr->getParameters()->setAttribute(scene::OBJ_LOADER_IGNORE_MATERIAL_FILES, true);
 
+	memset(&runData, 0, sizeof(runData));
```

Comment:
The compiler will likely inline the `memset` into a few store 0 instructions anyway